testing/src/main/java/org/oppia/android/testing/story/StoryProgressTestHelper.kt

```diff
@@ -327,16 +327,73 @@ class StoryProgressTestHelper @Inject constructor(
   }
 
   /**
-   * Marks the second chapter of test topic 0 story 0 as recently played. For specifics on
-   * parameters and nuances, see: [markRecentlyPlayedTestTopic0Story0Exp0].
+   * Marks the first chapter of test topic 0 story 0 as in progress saved. Note that this may require
+   * completing prerequisite chapters before the chapter can be marked as a prerequisite. See
+   * [markCompletedTestTopic0Story0Exp0] for specifics on the parameters passed to this method, and
+   * any other nuances.
    */
-  fun markRecentlyPlayedTestTopic0Story0Exp1(
+  fun markInProgressSavedTestTopic0Story0Exp0(
+    profileId: ProfileId,
+    timestampOlderThanOneWeek: Boolean
+  ) {
+    recordChapterAsInProgressSaved(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_0,
+      TEST_EXPLORATION_ID_2,
+      timestampOlderThanOneWeek
+    )
+  }
+
+  /**
+   * Marks the first chapter of test topic 0 story 0 as in progress not saved. Note that this may require
+   * completing prerequisite chapters before the chapter can be marked as a prerequisite. See
+   * [markCompletedTestTopic0Story0Exp0] for specifics on the parameters passed to this method, and
+   * any other nuances.
+   */
+  fun markInProgressNotSavedTestTopic0Story0Exp0(
```

Comment:
Add description of arguments in kdoc using  `@param `

Reply:
done